### ISSUE TITLE
Eristalis lineata Wahlberg, 1843 is a synonym of Cheilosia morio, but should not have its own site. The pictures and distribution show that of Eristalis lineata Harris, 1776!

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#4532]
**Eristalis lineata Wahlberg, 1843 is a synonym of Cheilosia morio, but should not have its own site. The pictures and distribution show that of Eristalis lineata Harris, 1776!**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/af83d0394bb2bf19f6560a6f8e395dc4:63f3d14510d5dabf452c29149e0ae95e5fb6d0da8d8d20684a60e50d50412cc451e1311d7b2a9611c140822b84eb37311111ab7b089e84d376d0748782c1d57367600b2a8f24469b0cd2d057ac5b2756600799a265fadb5ddc0c3f30c839abb363a7a3dafa4ecf6433b7699b96611c9ec2429c5b716e219b91097a994a416b881d284c6127277301b775fcd44b59790b) - [Send email](https://www.gbif.org/api/feedback/user/mailto/af83d0394bb2bf19f6560a6f8e395dc4:63f3d14510d5dabf452c29149e0ae95e5fb6d0da8d8d20684a60e50d50412cc451e1311d7b2a9611c140822b84eb37311111ab7b089e84d376d0748782c1d57367600b2a8f24469b0cd2d057ac5b2756600799a265fadb5ddc0c3f30c839abb363a7a3dafa4ecf6433b7699b96611c9ec2429c5b716e219b91097a994a416b881d284c6127277301b775fcd44b59790b)
System: Chrome 108.0.5359 / Windows 10.0.0
Referer: https://www.gbif.org/species/1540234
Window size: width 960 - height 468
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-01-17T10:50:06.411Z',to:'2023-01-17T10:56:06.411Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2023-01-17T10:50:06.411Z',to:'2023-01-17T10:56:06.411Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: INFO

**Comment by ManonGros** [comment]
It looks like most of the records are indeed Eristalis lineata Harris, 1776 but for some reason, they are matched to Eristalis lineata Wahlberg, 1843.
I am not sure why, is ir because of the parenthesis?

Field | Value
-- | -- 
Scientific name | Eristalis lineata 
Scientific name authorship |  (Harris, 1776)

- With the parenthesis, we don't match to the correct entry: https://api.gbif.org/v1/species/match?verbose=true&name=Eristalis%20lineata%20(Harris,%201776)
- Without the parenthesis, it seems ok https://api.gbif.org/v1/species/match?verbose=true&name=Eristalis%20lineata%20Harris,%201776

Is is the expected behaviour @mdoering ? Should I ask the publisher to remove parenthesis?

Issue partially relates to this one: https://github.com/gbif/portal-feedback/issues/4531

**Comment by mdoering** [comment]
Yes, parenthesis are rather important as we compare combination authors and basionym authors separately. With brackets there is no combination author to compare so it matches.

In fact both names match equally, both scoring 97. The algorithm then just picks one rather randomly as they are both synonyms of the same accepted name - i.e. ultimately it doesnt matter which is picked. It will be different if they are not synonyms of the same accepted name.

**Comment by mdoering** [comment]
I guess we should improve the matching to also try to match authorships ignoring parenthesis, scoring just a little lower as full matches.

**Comment by mdoering** [comment]
I have implemented that, so soon we'll match both with and without parenthesis to Harris not Wahlberg
